src/btree/bt_cursor.c

```diff
@@ -362,13 +362,15 @@ __wt_cursor_valid(WT_CURSOR_BTREE *cbt, WT_UPDATE **updp, bool *valid)
  *	Column-store search from a cursor.
  */
 static inline int
-__cursor_col_search(
-    WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_REF *leaf)
+__cursor_col_search(WT_CURSOR_BTREE *cbt, WT_REF *leaf, bool *leaf_foundp)
 {
 	WT_DECL_RET;
+	WT_SESSION_IMPL *session;
 
-	WT_WITH_PAGE_INDEX(session,
-	    ret = __wt_col_search(session, cbt->iface.recno, leaf, cbt, false));
+	session = (WT_SESSION_IMPL *)cbt->iface.session;
+	WT_WITH_PAGE_INDEX(
+		session, ret = __wt_col_search(
```

Comment:
Nit: You want to have 4 spaces here like before.

Reply:
done.